open last opened scene on start up

alonss0 opened the pull request:
This task is in draft as left to check or save the current scene index and load it when option enable on settings, also would be nice to change option text field to checkbox.

alonss0 committed:
CU-861m9bb3x first task commit

creating option on settings and calling function on awake with the value from playerprefs, left to check which is the last opened scene and calling it when option enabled.

MGH-Automator commented:
Task linked: [CU-861m9bb3x open last opened scene on start up](https://app.clickup.com/t/861m9bb3x)

alonss0 committed:
already saving the scene index, bug when trying to set it as current scene

alonss0 committed:
ready for review

setting the last opened scene index as current scene and modifying test as required for compile
